tests/core/block/test_block_manager.py

```diff
@@ -3,6 +3,8 @@
 
 import pytest
 
+pytestmark = pytest.mark.cpu_test
```

Comment:
`pre-commit` seems unhappy with the mark being set before the rest of the imports. Does it need to be up here to prevent potentially CUDA specific imports happening on the CPU machines?